source/usage-examples/watch.txt

```diff
@@ -4,7 +4,53 @@ Watch for Changes
 
 .. default-domain:: mongodb
 
-You can keep track of changes to data in MongoDB, such as changes to a
-collection, database, or deployment, by opening a **change stream**. A change
-stream allows applications to watch for changes to data and react to them.
-The change stream returns **change event** documents when changes occur.
+You can open a change stream on a ``MongoCollection``,
+``MongoDatabase``, or ``MongoClient`` by using the ``Watch()`` method.
+
+The following example passes an aggregation pipeline to the ``Watch()``
+method, which opens a change stream on the ``haikus`` collection and
+prints inserted documents:
+
+.. include:: /includes/usage-examples/run-example-tip.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/watch.go
+   :start-after: begin watch
+   :end-before: end watch
+   :emphasize-lines: 3
+   :language: go
+   :dedent:
+
+Click `here <{+example+}/watch.go>`__ to see a fully runnable example.
+
+Expected Result
+---------------
+
+After you run the preceding example, run the :doc:`Insert a
+Document usage example </usage-examples/insertOne>` in a different
+shell. When MongoDB completes the insert operation, you should see
```

Comment:
```suggestion
shell. After you run the other example, you should see
```
Suggestion: I think this should refer to the other task as a "usage example" both times. 

I also think having the subject switch from "you" to "mongodb" makes it hard to understand what I as the reader have to do to make the example work. I think the subject should be "you" in both sentences, and the action should be what the reader has to do: "run the example" (I think mentioning insert operations adds a new level of complexity that does not need to be present).